app/javascript/views/super_admin/notifications/edit.js

```diff
@@ -1,5 +1,39 @@
+<<<<<<< HEAD
+import { Tinymce } from '../../../utils/tinymce';
+// add the info on selecting the check from notification suitable
+=======
 import { Tinymce } from '../../../utils/tinymce.js.erb';
 
+//add the info on selecting the check from notification suitable
+>>>>>>> 8cebbdfe52128dbb2af7c3e1f7f1407be62795c4
```

Comment:
Looks like there's some lingering merge-conflicts in this bit.  Just take the latter section as it has the tinymce changes I'd made in development

Reply:
Ignore this, you've already made the change